### ISSUE TITLE
attrs 17.3.0 exception "ValueError: Cell is empty"

### ISSUE THREAD

**Issue by naomifox** [biocommons/hgvs#473]
With `attrs==17.3.0` and `hgvs==1.1.0.post1`, an exception is getting raised when building the RefTranscriptData class.

I temporarily solved the issue by installing `attrs==17.2.0`

```
In [1]: import hgvs.parser

In [2]: hgvs_c = 'NM_001637.3:c.1582G>A'

In [3]: hp = hgvs.parser.Parser()

In [4]: var_c = hp.parse_hgvs_variant(hgvs_c)

In [5]: import hgvs.dataproviders.uta

In [6]: import hgvs.assemblymapper

In [7]: hdp = hgvs.dataproviders.uta.connect()

In [8]: am = hgvs.assemblymapper.AssemblyMapper(hdp,
   ...: assembly_name='GRCh37', alt_aln_method='splign',
   ...: replace_reference=True)

In [9]: var_p = am.c_to_p(var_c)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/variant2pubmed/bin/apishell.py in <module>()
----> 1 var_p = am.c_to_p(var_c)

/usr/local/lib/python2.7/site-packages/hgvs/assemblymapper.pyc in c_to_p(self, var_c)
    115
    116     def c_to_p(self, var_c):
--> 117         var_out = super(AssemblyMapper, self).c_to_p(var_c)
    118         return self._maybe_normalize(var_out)
    119

/usr/local/lib/python2.7/site-packages/hgvs/variantmapper.pyc in c_to_p(self, var_c, pro_ac)
    294
    295         @attr.s(slots=True)
--> 296         class RefTranscriptData(object):
    297             transcript_sequence = attr.ib()
    298             aa_sequence = attr.ib()

/usr/local/lib/python2.7/site-packages/attr/_make.pyc in wrap(cls)
    635             builder.add_init()
    636
--> 637         return builder.build_class()
    638
    639     # maybe_cls's type depends on the usage of the decorator.  It's a class

/usr/local/lib/python2.7/site-packages/attr/_make.pyc in build_class(self)
    370         """
    371         if self._slots is True:
--> 372             return self._create_slots_class()
    373         else:
    374             return self._patch_original_class()

/usr/local/lib/python2.7/site-packages/attr/_make.pyc in _create_slots_class(self)
    460                 continue
    461             for cell in closure_cells:
--> 462                 if cell.cell_contents is self._cls:
    463                     set_closure_cell(cell, cls)
    464

ValueError: Cell is empty
```

**Comment by GerardGH** [comment]
Same problem here

**Comment by reece** [comment]
Fixed in 1.1.1, now available on pypi. Full changelog here:
http://hgvs.readthedocs.io/en/stable/changelog/1.1/1.1.1.html
